docs/source/broadcasting.md

```diff
@@ -13,6 +13,200 @@ and [have the node running](node.html).
+
+One way to make the verifier endpoint accessible to the broadcaster without making it publicly accessible is to create an SSH tunnel from the broadcaster machine to the verifier machine.
+
+Create an SSH tunnel with the following command run from the broadcaster machine:
+
+```
+ssh -N -L 5000:127.0.0.1:5000 -i <SSH_KEY_FILE> <USER>@<HOSTNAME>
+```
+
+Note that the command will not output anything if it is successfuly run.
+
+One way to make the verifier's volume accessible over the network is to mount the verifier's volume on the broadcaster machine over an SSH connection using [sshfs](https://github.com/libfuse/sshfs).
+
+Mount the verifier's volume on the broadcaster machine by running the following command on the broadcaster machine:
+
+```
+sshfs -o IdentityFile=<SSH_KEY_FILE> <USER>@<HOSTNAME>:<FOLDER_ON_VERIFIER> <FOLDER_ON_BROADCASTER>
+```
+
+After the SSH tunnel is setup and the remote volume is mounted, run the following command to start the broadcaster:
+
+```
+livepeer -network rinkeby -broadcaster -verifierUrl http://localhost:5000/verify -verifierPath <FOLDER_ON_BROADCASTER>
+```
+
+The broadcaster also supports sharing segment data with a verifier using external cloud storage providers such as Amazon's S3 or Google's GCS. Documentation on using external cloud storage providers will be added in the future.
+
+## Running an Ethereum node
+
+By default `livepeer` will use Infura as the ETH RPC provider. If you would like to run your own ETH node you can use `geth` (the Rinkeby testnet is only supported by `geth`) which can be installed using the instructions on the [installing geth page](https://geth.ethereum.org/docs/install-and-build/installing-geth). 
+
+Run `geth`:
+
+```
+$ geth -rinkeby -rpc -rpcapi eth,net,web3
+```
+
+If `geth` is running on a different machine than `livepeer` you will have to specify the `-rpcaddr` flag to indicate the interface to listen on.
+
+Wait until `geth` is fully synced with the latest block on the Rinkeby testnet. You can check if `geth` is done syncing by using the Geth Javascript Console:
+
+```
+$ geth attach http://localhost:8545
+Welcome to the Geth JavaScript console!
+
+instance: Geth/v1.9.0-stable-52f24617/linux-amd64/go1.12.7
+coinbase: 0x0161e041aad467a890839d5b08b138c1e6373072
+at block: 583 (Wed, 23 Oct 2019 17:41:00 EDT)
+ modules: debug:1.0 eth:1.0 miner:1.0 net:1.0 personal:1.0 rpc:1.0 txpool:1.0 web3:1.0
+
+> eth.syncing
+false
+```
+
+In order to connect to your own ETH node, you will need to set the `-ethUrl` flag on your `livepeer` node to the ETH node's RPC URL.
+
+For example, you could use the following command to connect an orchestrator to an ETH node running at `localhost:8545`.
+
+```
+$ livepeer -network rinkeby -orchestrator -pricePerUnit 1 -ethUrl http://localhost:8545
+```
+
+## Migrating to Streamflow
```

Comment:
Since livepeer/go-livepeer#1299 is merged let's update the description to say something like:

"If you are upgrading from a go-livepeer < v0.5.3 you do *not* need to take any additional action - the node will create a fresh DB that is compatible with the changes included in the Streamflow protocol upgrade."